src/ParagonProvider/README.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'ParagonProvider'
+type: 'component'
+components:
+- ParagonProvider
+- ParagonContext
+categories:
+- Theming
+status: 'Stable'
+designStatus: 'Done'
+devStatus: 'Done'
+notes: |
+
+---
+
+Use the `ParagonProvider` at the root of a React app to set the Paragon CSS-in-JS theme and locale.
```

Comment:
Should there be a note here or elsewhere that this is, in fact, required in order to use associated elements?

Reply:
Good call, I'll add to the "getting started and installation" page

Reply:
Updated